selfdrive/controls/lib/latcontrol_indi.py

```diff
@@ -64,6 +64,8 @@ def update(self, active, v_ego, angle_steers, angle_steers_rate, steer_override,
     else:
       self.angle_steers_des = path_plan.angleSteers
       self.rate_steers_des = path_plan.rateSteers
+      if not steer_override:
```

Comment:
The fast angle offset value becomes disruptive during steer_override, so it's best to NOT update the bias during the override